source/crypto.c

```diff
@@ -29,6 +29,29 @@ static void s_hmac_destructor(PyObject *hmac_capsule) {
     aws_hmac_destroy(hmac);
 }
 
+PyObject *aws_py_sha1_new(PyObject *self, PyObject *args) {
+    (void)self;
+    (void)args;
+
+    struct aws_allocator *allocator = aws_py_get_allocator();
+
+    struct aws_hash *sha1 = aws_sha1_new(allocator);
+
+    if (!sha1) {
+        return PyErr_AwsLastError();
+    }
+
+    PyObject *capsule = PyCapsule_New(sha1, s_capsule_name_hash, s_hash_destructor);
+
+    /* Why do we return NULL instead of PyErr */
```

Comment:
Is this still a question we have?  Can we remove this comment?

Reply:
trivial remove. Same question you had in the above comment.